### DISCUSSION TITLE
Reconnect

### DISCUSSION THREAD

**Discussion by andre-merzky** [radical-cybertools/radical.pilot#2414] [Ideas]
I am closing a couple of issues which ultimately all require the same thing: reconnect on network drops and session timeouts (for MongoDB and ssh).  

It is unclear when exactly this will be implemented.  The ssh reconnection should probably happen at saga level; the MongoDB reconnect needs work on the RP level, to ensure that no state updates are lost while the connection is down.

**Comment by mturilli** [comment]
I am assigning this to v2 and closing it.
